webpack/react_app/components/TargetingHosts/HostItem/HostItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ActionButtons } from 'foremanReact/components/common/ActionButtons/ActionButtons';
+import HostStatus from '../HostStatus/HostStatus';
+
+const HostItem = ({ name, link, status, actions }) => {
+  const renderName = () => {
+    if (link) {
+      return <a href={link}>{name}</a>;
```

Comment:
it may be a good thing to have css class/html id added to everything that is clickable, so we can write tests against it

Reply:
I added `id` selector to parent `<tr>` element so other child `<td>` can be easily selected as well.

```
<tr id="targeting-host-zzzz113.example.com">
...
</tr>
```